src/components/pages/HomePage/Features/featuresList.ts

```diff
@@ -18,7 +18,7 @@ const featuresList = [
   {
     picture: MessagingAndStreaming,
     title: 'Low-latency messaging and streaming',
-    text: 'Acknowledge messages individually (RabbitMQ style) or cumulative per partition (i.e., offset-like). Enables use cases such as distributed work queues or order-preserving data streams at very large scales (hundreds of nodes) and low latency (<5ms).',
+    text: 'Acknowledge messages individually (RabbitMQ style) or cumulative per partition (i.e., offset-like). Enables use cases such as distributed work queues or order-preserving data streams at very large scales (hundreds of nodes) and low latency (<10ms).',
```

Comment:
This is to align with https://github.com/apache/pulsar-site/pull/560#issuecomment-1553930425